tests/loggers/test_loggers.py

```diff
@@ -0,0 +1,152 @@
+# --------------------------------------------------------------------------------------
+# Part of the interTwin Project: https://www.intertwin.eu/
+#
+# Created by: Anna Elisa Lappe
```

Comment:
If I well remember I created this. If I am right, you can anyways add yourself to the credit ;)

Reply:
Omg haha sorry, copy paste got me 🙈